config.toml

```diff
@@ -0,0 +1,74 @@
+checkUpdate = true
+debug = false
+interval = 6
+cleanInterval = 3_600
+banTime = 86_400
+timeout = 6
+logToFile = true
+
+clientType = ""
+clientURL = "http://qbittorrent"
+clientUsername = ""
+clientPassword = ""
+
+useBasicAuth = false
+skipCertVerification = false
+
+blockListFile = [
+    "blockList.json",
+    # "blockList-Optional.json"
+]
+blockListURL = [
+    "https://bta.iaalai.cn/qBittorrent-ClientBlocker/blockList.json",
+    "https://cdn.jsdelivr.net/gh/Simple-Tracker/qBittorrent-ClientBlocker@dev/blockList.json",
+]
+portBlockList = []
+ipBlockList = ["ipBlockList.txt"]
```

Comment:
This is a mistake in the previous default configuration, it should be ipBlockListFile.

Reply:
Fixed. The name in `config.json` also changed.